app/Services/Apis/PaymentGateways/StripeApi.php

```diff
@@ -86,6 +86,25 @@ public function setWebHookSecretKey(string $webhook_secret_key):void{
         $this->webhook_secret_key = $webhook_secret_key;
     }
 
+    public function getCreditCardInfo(array $payload): array
+    {
+        $charges = $payload['charges']['data'];
```

Comment:
check this keys exists at payload before doing evaluation

Reply:
done